contrib/patterns/_grid-media.scss

```diff
@@ -0,0 +1,52 @@
+$medium-screen: 1000px;
```

Comment:
Somehow this ended up commenting on the wrong line. Posting again:

It's a little more verbose, but do you not like the normal styleguide way?

```
.grid-column--media-3-to-6 {
  @include grid-column(3);

  @include grid-media($custom-neat-grid) {
    @include grid-column(6);
  }

  @include grid-media($specific-neat-grid) {
    @include grid-column(6);
  }

  @include grid-media($print-neat-grid) {
    @include grid-column(6);
  }

  &::before {
    content: "#{map-get($neat-grid, media)}";

    @include grid-media($custom-neat-grid) {
      content: "#{map-get($custom-neat-grid, media)}";
    }

    @include grid-media($specific-neat-grid) {
      content: "#{map-get($specific-neat-grid, media)}";
    }

    @include grid-media($print-neat-grid) {
      content: "#{map-get($print-neat-grid, media)}";
    }
  }
}
```

I don't really care about breaking style here, but it might be good to prevent Hound complaining about future changes